assets/js/admin/meta-boxes-order.js

```diff
@@ -549,45 +549,31 @@ jQuery( function ( $ ) {
 		},
 
 		add_fee: function() {
```

Comment:
I would say that the original behavior here is okay, and instead the focus should be on resolving the server-side parsing in: https://github.com/woocommerce/woocommerce/blob/1d1413b4ee669cd37ee025c93972ce785a10a0b9/includes/class-wc-ajax.php#L995

It's worth noting that there does seem to be inconsistencies across WC in whether we respect the currency symbol or not. In this case, we should probably just remove the default currency symbol from the string before converting it to a float value.